### DISCUSSION TITLE
bug? in using bs/alert + markdownify

### DISCUSSION THREAD

**Discussion by evandde** [hbstack/.github#16] [Q&A] [answered]
When I use bs/alert shortcode and markdownify, hyperlink is converted in weird format.

```markdown
{{< bs/alert info >}}
{{% markdownify %}}
Hello!

This is a [hyperlink](https://hbstack.dev/)
{{% /markdownify %}}
{{< /bs/alert >}}
```

And then the result seems like

> Hello!

> This is a [target="_blank" rel=“noopener noreferrer”>hyperlink](https://hbstack.dev/)

`target="_blank" rel=“noopener noreferrer”>` should not appear.

~Can I fix it?~

Maybe... I must use `{{< bs/alert-link "an example link" "#alert-link" >}}`..?

But when I use `alert-link`, the link opens on same page. 

Can I change it to open on new tab page? (target="_blank")

**Comment by razonyang** [comment, marked as answer]
Use `<>` instead of `%%` should work, since the output contains RAW HTML.

```markdown
{{< bs/alert info >}}
{{< markdownify >}}
**Hello!**

This _is_ a [hyperlink](https://hbstack.dev/)
{{< /markdownify >}}
{{< /bs/alert >}}
```
